augur/mask.py

```diff
@@ -231,8 +231,7 @@ def run(args):
             print("ERROR: File {} does not exist!".format(args.mask_file))
             sys.exit(1)
         if os.path.getsize(args.mask_file) == 0:
-            print("ERROR: {} is an empty file.".format(args.mask_file))
-            sys.exit(1)

```

Comment:
The pytest failure isn't very clear, but it fails on this test:

https://github.com/nextstrain/augur/blob/2810abe774e6b07d998d96cf36d345b2acd95c1c/tests/test_mask.py#L257-L261

Failing there makes sense because a `SystemExit` is no longer raised. However, by warning and letting `augur mask` continue to run with an empty mask file, there is a new error downstream:

```
augur/mask.py:244: in run
    mask_sites.update(load_mask_sites(args.mask_file))
augur/utils.py:531: in load_mask_sites
    mask_sites = read_bed_file(mask_file)
augur/utils.py:479: in read_bed_file
    bed = pd.read_csv(bed_file, sep='\t', header=None, usecols=[1,2],
…/site-packages/pandas/util/_decorators.py:211: in wrapper
    return func(*args, **kwargs)
…/site-packages/pandas/util/_decorators.py:331: in wrapper
    return func(*args, **kwargs)
…/site-packages/pandas/io/parsers/readers.py:950: in read_csv
    return _read(filepath_or_buffer, kwds)
…/site-packages/pandas/io/parsers/readers.py:605: in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
…/site-packages/pandas/io/parsers/readers.py:1442: in __init__
    self._engine = self._make_engine(f, self.engine)
…/site-packages/pandas/io/parsers/readers.py:1753: in _make_engine
    return mapping[engine](f, **self.options)
…/site-packages/pandas/io/parsers/c_parser_wrapper.py:79: in __init__
    self._reader = parsers.TextReader(src, **kwds)

>   ???
E   pandas.errors.EmptyDataError: No columns to parse from file

pandas/_libs/parsers.pyx:554: EmptyDataError
```

The downstream code should be updated to support empty mask files.